packages/create/src/validation/validation.ts

```diff
@@ -0,0 +1,33 @@
+import { existsSync } from 'fs';
+import type { Editor } from 'mem-fs-editor';
+import { join } from 'path';
+import type { EditorWithDump } from '../types';
+
+/**
+ * Validate base path of app, throw error if file is missing.
+ *
+ * @param basePath - base path of the app, where package.json and ui5.yaml resides
+ */
+export function validateBasePath(basePath: string): void {
```

Comment:
Is it possible that this one is outdated e.g. hardcoded `webapp` path but the consumer of this function, searches after validation for the webapp path

Reply:
function `validateBasePath()` just checks whether we have a `webapp` path or a `ui5.yaml` file, which could contain a custom mapping for webapp. It doesn't resolve the webapp path. I saw we use a similar approach in other writers.
I would not combine the two functionalities get webapp path and validate base path together, I have cases where I do not need the webapp path (remove mockserver config), but I still would like to do a basic validation to check whether the folder is a valid basePath aka app root.